patch (adl): [considerations] access tiers (#1335)
MicrosoftDocs/cloud-adoption-framework: docs/scenarios/cloud-scale-analytics/best-practices/data-lake-key-considerations.md

- **Cold tier:** Optimized for storing data that is infrequently accessed or modified. Data is stored for at least 90 days. The cold tier has lower storage costs and higher access costs compared to the cool tier.
- **Archive:** Optimized for storing data that's rarely accessed. The data is stored for at least 180 days with flexible latency requirements, on the order of hours.

> [!IMPORTANT]

> > There are no reliability, security, operational excellence performance efficiency tradeoffs between the various online access tiers, leaving the choice of an online tier to be a financial decision, per-blob, based on workload access data size, operational interactions, and time before the blob is deleted. Select the correct tier, per blob, based on [a calculation of the above factors](https://azure.github.io/Storage/docs/application-and-user-data/code-samples/estimate-block-blob). For more information, please take a look at the [Plan and manage costs for Azure Blob Storage ](/azure/storage/common/storage-plan-manage-costs).

Reviewer: ```suggestion
> There are no reliability, security, operational excellence, or performance efficiency tradeoffs between the various online access tiers, which leaves the choice of an online tier to be a financial decision, per blob, based on workload access data size, operational interactions, and time before the blob is deleted. Select the correct tier, per blob, based on [a calculation of the preceding factors](https://azure.github.io/Storage/docs/application-and-user-data/code-samples/estimate-block-blob). For more information, see [Plan and manage costs for Azure Blob Storage ](/azure/storage/common/storage-plan-manage-costs).
```
